lisp/ess-custom.el

```diff
@@ -2587,14 +2587,16 @@ This variable has no effect. Customize
   (append ess-RS-constants
           '("T" "F")))
 
-(defvar ess-R-keywords
-  ;; "Reserved Words" -- part 2 --
-  '("while" "for" "in" "repeat" "if" "else" "switch" "break" "next" "function"
-    ;; note that these are *NOT* reserved words in R:
-    "return" "message" "warning" "stop"))
+(defconst ess-r--keywords
+  '("in" "else" "break" "next"))
```

Comment:
I don't think we should changer/remove this. It's a very old variable and people are probably customizing it. I would also not down-case it for now. Either downcase all such variables (and alias) or none. 

Why don't you just create a new defconst for what you need?

Reply:
> Why don't you just create a new defconst for what you need?

We have to split this variable in two parts to fix its fontification. Maybe we can add an alias so the old symbol points to the new function-like keywords?

I _think_ it's better to gradually move towards a consistent naming scheme when we make modifications.

Reply:
> Maybe we can add an alias so the old symbol points to the new function-like keywords?

Done. Should we obsolete the old variable?

> I think it's better to gradually move towards a consistent naming scheme when we make modifications.

Otherwise convergence will likely never happen.

Reply:
>  it's better to gradually move towards 

That creates confusion and annoyance. At least it should be done within blocks of related functionality (fontlock, indentation, interaction etc).

Reply:
This R-r dichotomy is generally not bothering me as much as it does you. I think as long as within related groups of functions notation is consistent, it's not that bad.